data/json/mapgen/veterinarian.json

```diff
@@ -0,0 +1,203 @@
+[
+{
+        "type": "vehicle_group",
+        "id": "VETS",
+        "vehicles": [
+            [ "car", 300 ],
+            [ "pickup", 100 ],
+			[ "suv", 50 ]
+        ]
+    },{
+        "type": "item_group",
+        "id": "vet_softdrug",
+        "items": [
+            ["bandages", 50],
+            ["1st_aid", 10],
+			["saline", 20],
+			["gummy_vitamins", 15],
+			["nyquil", 10],
+			["disinfectant", 15],
+			["vaccine_shot", 10],
+			["protein_powder", 30],
+			["smoxygen_tank", 20],
+			["elec_hairtrimmer", 10],
+			["survnote", 5],
+			["mask_dust", 65],
+			["glasses_safety", 10],
+			["towel", 50],
+			["scissors", 50],
+			["fungicide", 10],
+			["soap", 20],
+			["chem_hydrogen_peroxide", 10],
+			["canister_empty", 5],
+			["stethoscope", 5],
+			["scalpel", 55],
+			["gloves_medical", 50],
+			["medical_gauze", 25],
+			["quikclot", 10],
+			["hacksaw", 5],
+			["gloves_rubber", 25],
+			["knife_butcher", 5],
+			["machete", 5],
+			["rag_bloody", 10],
+			["bag_plastic", 20],
+			["string_6", 5]
+        ]
+    },{
+        "type": "item_group",
+        "id": "vet_utility",
+        "items": [
+			["dog_whistle", 10],
+			["pet_carrier", 30],
+			["dogfood", 30],
+			["catfood", 30],
+			["towel", 20],
+			["soap", 10],
+			["gloves_medical", 20],
+			["rag", 15],
+			["rag_bloody", 2],
+			["bag_plastic", 20],
+			["jerky", 40]
+        ]
+    },{
+        "type": "item_group",
+        "id": "vet_hardrug",
+        "items": [
+            ["syringe", 10],
+            ["diazepam", 10],
+			["antifungal", 30],
+			["antiparasitic", 30],
+			["antibiotics", 20],
+			["thorazine", 7],
+			["morphine", 50],
```

Comment:
I have to get an idea of what drugs an average vet would carry that would actually affect humans in a similar enough manner to be useful.